connect/runtime/src/test/java/org/apache/kafka/connect/runtime/AbstractHerderTest.java

```diff
@@ -77,12 +77,7 @@ public void connectorStatus() {
                 .andReturn(Collections.singletonList(
                         new TaskStatus(taskId, AbstractStatus.State.UNASSIGNED, workerId, generation)));
         EasyMock.expect(worker.getPlugins()).andStubReturn(plugins);
-        final Connector conn;
-        try {
-            conn = TestSourceConnector.class.newInstance();
-        } catch (InstantiationException | IllegalAccessException e) {
-            throw new RuntimeException("Couldn't create connector", e);
-        }
+        final Connector conn = new TestSourceConnector();
         EasyMock.expect(plugins.newConnector(connector)).andReturn(conn);
 
         replayAll();
```

Comment:
Seems like the connector type should be checked in the remainder of this unit test.